src/api/c/plot.cpp

```diff
@@ -105,3 +105,25 @@ af_err af_draw_plot(const af_window wind, const af_array X, const af_array Y, co
     return AF_ERR_NO_GFX;
 #endif
 }
+
+af_err af_draw_plot(const af_window wind, const af_array X, const af_array Y, const af_cell* const props)
+{
+    return plotWrapper(wind, X, Y, props);
+}
+
+af_err af_draw_scatter(const af_window wind, const af_array X, const af_array Y, const af_cell* const props, const af::markerType af_marker)
+{
+    fg::MarkerType fg_marker;
+    switch(af_marker){
+        case AF_MARKER_NONE: fg_marker = fg::FG_NONE; break;
+        case AF_MARKER_POINT: fg_marker = fg::FG_POINT; break;
+        case AF_MARKER_CIRCLE: fg_marker = fg::FG_CIRCLE; break;
+        case AF_MARKER_SQUARE: fg_marker = fg::FG_SQUARE; break;
+        case AF_MARKER_TRIANGLE: fg_marker = fg::FG_TRIANGLE; break;
+        case AF_MARKER_CROSS: fg_marker = fg::FG_CROSS; break;
+        case AF_MARKER_PLUS: fg_marker = fg::FG_PLUS; break;
+        case AF_MARKER_STAR: fg_marker = fg::FG_STAR; break;
+        default: fg_marker = fg::FG_NONE; break;
+    }
```

Comment:
Move this switch statement into a helper function, may be move it to `graphics_common.[hpp/cpp]` source files, so that is accessible for any other functions in future.